include/tf2_2d/rotation.hpp

```diff
@@ -0,0 +1,277 @@
+/*
+ * Software License Agreement (BSD License)
+ *
+ *  Copyright (c) 2019, Locus Robotics
```

Comment:
```suggestion
 *  Copyright (c) 2022, Locus Robotics
```

Reply:
I'm not sure about this one. I renamed the file from `rotation.h` to `rotation.hpp` (motivated by making the linter recognize it as a C++ file instead of a C file), but the content isn't new.

Reply:
Hmm, how about on the redirection header instead?

Reply:
Bumped year on redirection headers in 63edaa5